src/pages/getting-started/hello-world.mdx

```diff
@@ -251,6 +261,7 @@ console.log(`1 ADA locked into the contract
     Tx ID: ${txLock}
     Datum: ${datum}
 `);
+// -- The tADA amount above in `console.log` can be arbritary depending on how much you wish to transact so feel free to edit your code accordingly.
```

Comment:
```suggestion
```

Reply:
Not really, we also assume 1 ADA after, and for the sake of not introducing inconsistency and making the tutorial possibly _fail_ if people get too adventurous, let's not suggest that.